packages/core-contracts/contracts/mocks/proxy/UniversalProxyImplementationMockB.sol

```diff
@@ -0,0 +1,48 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../../proxy/UniversalProxyImplementation.sol";
+
+contract UniversalProxyImplementationMockB is UniversalProxyImplementation {
+    uint private _a;
+    bytes32 private _b;
```

Comment:
set to bytes32 instead of structured string to have more control over the memory size of the slot.